Gordon360/Models/ViewModels/RecIM/ActivityCreatedViewModel.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Gordon360.Models.CCT;
+using System;
+using System.Collections.Generic;
+
+namespace Gordon360.Models.ViewModels.RecIM
+{
+    public class ActivityCreatedViewModel
```

Comment:
I'm wondering if it is necessary to get so granular with your view model here. Is there a need to return a different object type on activity creation vs on activity update (or activity delete for that matter).

What is the need for having both `ActivityViewModel` and `ActivityCreatedViewModel`? I see that the former has the Series and Team data, but where is it necessary to distinguish between the two in return data.

If there is a reason for both to exist, I think the names should be more descriptive for what information they contain rather than what they may be used for. In my mind, `ActivityExtendedViewModel` and `ActivityViewModel` make sense, but there are a bunch of variations that can work well.

Perhaps this is something that @EjPlatzer would like to weigh in on. The question in my mind is "should we have a naming convention to distinguish view models that have a bunch of related object arrays from those that are just the simple object?".

Reply:
My primary argument for _____CreatedViewModel was so that I could have a clear type to return to the typescript service. I did not want to have a bunch of nullable fields within each type but this is one of the things that I think I brought up in our last conversation.